coala_quickstart/generation/Bears.py

```diff
@@ -200,38 +200,48 @@ def get_non_optional_settings_bears(bears):
 
 def remove_unusable_bears(bears, unusable_bears):
```

Comment:
I think based on the new changes, a better name could be `filter_unsuable_bears`

Reply:
Oh, you are right! I didn't thought about that! Makes more sense now! Thank you!